gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/templates/alert_responseTime.html

```diff
@@ -5,6 +5,7 @@
       <p>
         Average response time over ${alert.conditions[0].threshold}ms during ${alert.conditions[0].duration} ${alert.conditions[0].timeUnit}.
       </p>
+      <p>${alert.description}</p>
```

Comment:
Description is not mandatory so, if not set, it will result an empty paragraph.

Should we better use a if statement ?